src/edit/handler/Edit.Poly.js

```diff
@@ -74,8 +95,158 @@ L.Edit.Poly = L.Handler.extend({
+					.on('dragend', this._onMarkerDragEnd, this)
+					.on('touchstart', this._onTouchStart, this)
+					.on('touchmove', this._onTouchMove, this)
+					.on('MSPointerMove', this._onTouchMove, this)
+					.on('touchend', this._onTouchEnd, this)
+					.on('MSPointerUp', this._onTouchEnd, this);
+
+				this._markerGroup.addLayer(this._moveMarker);
+			}
+		}
+	},
+
+	_releaseMarkers: function () {
+		this._moveMarker
+			.off('dragstart', this._onMarkerDragStart, this)
+			.off('drag', this._onMarkerDrag, this)
+			.off('dragend', this._onMarkerDragEnd, this)
+			.off('touchstart', this._onTouchStart, this)
+			.off('touchmove', this._onTouchMove, this)
+			.off('MSPointerMove', this._onTouchMove, this)
+			.off('touchend', this._onTouchEnd, this)
+			.off('MSPointerUp', this._onTouchEnd, this);
+
+		this._markerGroup.removeLayer(this._moveMarker);
+		delete this._moveMarker;
+
+		this._map.removeLayer(this._markerGroup);
+		delete this._markerGroup;
+
+		delete this._map;
+
+		this._poly.off('edit', this._onEdit, this);
+	},
+
+	_fireEdit: function () {
+		this._poly.edited = true;
+		this._poly.fire('edit');
+	},
+
+	_onEdit: function (e) {
+		if (this._moveMarker) {
+			var latlng = this._getMoveMarkerLatLng();
+
+			this._moveMarker.setLatLng(latlng);
+			this._moveMarker._origLatLng = latlng;
+		}
+	},
+
+	_onMarkerDragStart: function (e) {
+		var marker = e.target;
+		marker.setOpacity(0);
 
+		this._poly.fire('editstart');
+	},
+
+	_onMarkerDrag: function (e) {
+		var marker = e.target,
+			latlng = marker.getLatLng();
+
+		this._move(latlng);
+
+		this._poly.fire('editdrag');
+	},
+
+	_onMarkerDragEnd: function (e) {
+		var marker = e.target;
+		marker.setOpacity(1);
+
+		this._fireEdit();
+	},
+
+	_onTouchStart: function (e) {
+		var marker = e.target;
+		marker.setOpacity(0);
+
+		this._poly.fire('editstart');
+	},
+
+	_onTouchMove: function (e) {
+		var layerPoint = this._map.mouseEventToLayerPoint(e.originalEvent.touches[0]),
+			latlng = this._map.layerPointToLatLng(layerPoint);
+
+		this._move(latlng);
+
+		return false;
+	},
+
+	_onTouchEnd: function (e) {
+		var marker = e.target;
+		marker.setOpacity(1);
+
+		this._fireEdit();
+	},
+
+	_move: function (latlng) {
+		var moveMarker = this._moveMarker;
+
+		var latlngs = this._defaultShape();
+
+		var latMove = latlng.lat - moveMarker._origLatLng.lat;
+		var lngMove = latlng.lng - moveMarker._origLatLng.lng;
+
+		for (var i = 0; i < latlngs.length; ++i) {
+			latlngs[i].lat += latMove;
+			latlngs[i].lng += lngMove;
+		}
+
+		moveMarker.setLatLng(latlng);
+		moveMarker._origLatLng = latlng;
+
+		this._poly.redraw();
+		this.updateMarkers();
+
+		this._map.fire(L.Draw.Event.EDITMOVE, {layer: this._poly});
+	},
+
+	_getMoveMarkerLatLng: function () {
+
+		var latlngs = this._defaultShape();
+
+		var p1 = this._map.project(latlngs[0]);
+		var p2 = this._map.project(latlngs[1]);
+
+		return this._map.unproject(p1._multiplyBy(0.75)._add(p2._multiplyBy(0.25)));
```

Comment:
Thank you for reviewing.
My intention was to have a marker for movement to locate between the first and second marker (not to overlap with other markers).
But it seems something was going wrong.
I merged the latest and pushed it.
Thanks!.